src/bp/core/telemetry/stats/actions-stats.ts

```diff
@@ -61,7 +61,7 @@ export class ActionsStats extends TelemetryStats {
   }
 
   private async getFlowsWithActions(): Promise<ParsedFlow[]> {
-    const botIds = await this.botService.getBotsIds()
```

Comment:
The `botService` injected class instance isn't used anymore. Should you remove it?